docs_src/use-cases/loss-prevention/advanced.md

```diff
@@ -41,6 +41,17 @@ To find all supported classes by YOLOv8, you can find them in this file `src/ext
 CLASS_IDS="0,2"
 ```
 
+## Use of the Age Classification Pipeline
+
+To run an age classification pipeline, let's change some variables to the execution:
+
+!!! Example
+
+    ```bash
+    make PIPELINE_SCRIPT=age_recognition.sh RESULTS_DIR="../render_results" run-render-mode
+    ```
+
+The above command will execute a DLStreamer pipeline using a facial detection model for object detection, then running an age/gender classification model on the results of the facial detection model to output the gender and age of the faces in the input frame.
```

Comment:
you can mention which models are used here for facial detection and age/gender classification

Reply:
Do you think it would be better if I link to the OpenVINO model docs page or just mention the specific model names?

Reply:
you can link the openvino model repo